cloudbuild.yaml

```diff
@@ -48,7 +51,8 @@ steps:
     git clone git@github.com:openconfig/models-ci.git /go/src/github.com/openconfig/models-ci
     cd /go/src/github.com/openconfig/models-ci
     # Modify the major version to update models-ci version.
-    branch=$(git tag -l 'v8*' | sort -V | tail -1)
+    #branch=$(git tag -l 'v8*' | sort -V | tail -1)
+    branch=ocpyang-version #FIXME: remove
```

Comment:
Once https://github.com/openconfig/models-ci/pull/80
and
https://github.com/openconfig/oc-pyang/pull/53

are merged, I will remove this.